### PR TITLE
add actionlint script and fix linting errors

### PR DESCRIPTION
**Pull request by Mic92**
While it might be worth adding this in future as a separate CI step, it seems overkill given how often it would need to run on nixpkgs (could be circumvented by filtering, but it would still add a check line in each PR). This script is also easier to run local.

### CODE REVIEW

**Review by azuwis — CHANGES_REQUESTED** [review]

File: .github/workflows/lint-actions.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash actionlint shellcheck -I nixpkgs=../..
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+actionlint $SCRIPT_DIR/*.yml
```
[the remark this answers is not in the document]
Comment (Mic92) [reply .github/workflows/lint-actions.sh:4]: What if you are not in the repository?

[the remark this answers is not in the document]
Comment (Mic92) [reply .github/workflows/lint-actions.sh]: I did a combination of our two approaches.

File: .github/workflows/lint-actions.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash actionlint shellcheck -I nixpkgs=../..
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```
[the remark this answers is not in the document]
Comment (Mic92) [reply .github/workflows/lint-actions.sh:5]: See above.

**Review by azuwis — APPROVED** [review]

**Comment by infinisil** [comment]
@azuwis Would you mind also being registered as `.github/workflows` code owner, so that you get automated review requests for PRs? :D

**Pull request merged**